e2e/jail/jail_test.go

```diff
@@ -154,10 +179,63 @@ func (s *JailTestSuite) TestEventSignal() {
 	response, err := responseValue.ToString()
 	s.NoError(err, "cannot parse result")
 
-	expectedResponse := `{"jsonrpc":"2.0","result":true}`
+	expectedResponse := `{"result":true}`
 	s.Equal(expectedResponse, response)
 }
 
+// TestCallResponseOrder tests for problem in
+// https://github.com/status-im/status-go/issues/372
+func (s *JailTestSuite) TestCallResponseOrder() {
```

Comment:
May this test be more simple? If it tests call response order, why are other complicated things needed?

Reply:
@tiabc it actually tests specific case in the issue mentioned in the comment. It's not general requests/responses ordering test.

Reply:
I mean let's modify the function of the comment to be more specific that it's not just about call response order?

Reply:
Ah.. Any suggestion for better name? Function naming is limited by definition, one more reason to switch to BDD approach eventually.

Reply:
Just something like:
```
// TestCallResponseOrder tests exactly the problem from
// https://github.com/status-im/status-go/issues/372
```
?